config.json

```diff
@@ -1,5 +1,5 @@
 {
-"timeout": 100,
+"timeout": 1,
```

Comment:
is this timeout change intentional?

Reply:
In this case, it was more or less intentional. So far, the creation of all benchmarks has not been executed and I played around with the timeout limit for testing purposes. As a consequence, the actual number is not relevant right now and both `100` and `1` are equally "correct".

Reply:
Sounds reasonable to me 👍🏼